### DISCUSSION TITLE
Add more SignUp / SignIn Options for cloud service account

### DISCUSSION THREAD

**Discussion by SebasScript** [supabase/supabase#7102] [Questions] [answered]
Is there a plan to add plain email and password signup / signin for the cloud service account (https://app.supabase.com/)?
As at the moment there is only GitHub for login, if I would loose access to my GitHub account for whatever reason, I would loose access to all my cloud deployed projects which is a bit worrying. It would be great if there are more options and possibility to link those options for the account (Gitlab, Email and password, two factor auth for security etc. etc.)

**Comment by dshukertjr** [comment, marked as answer]
Hi @SebasScript! 

We don't have concrete timelines as of now, but this is a frequently requested feature, so we will most likely expand the login option to beyond Github login sometime in the (hopefully not so far) future 😄

**Reply by SebasScript** [reply]
thanks for the update, and good to know that its a frequently requested feature. Depending on 3rd party login provider for something import as cloud deployments and customer data is not great in the long run
